native/src/bin/killer.rs

```diff
@@ -0,0 +1,42 @@
+#[cfg(target_family = "windows")]
```

Comment:
I don't believe you need these on `use` statements. That isn't an explicit import like in js, but rather declaring a "shorthand" version.